odoo/http.py

```diff
@@ -2053,7 +2053,7 @@ def session_store(self):
     def get_db_router(self, db):
         if not db:
             return self.nodb_routing_map
-        return request.registry['ir.http'].routing_map()
+        return request.env['ir.http'].routing_map()
```

Comment:
Please explain why you changed this, as using `env` instead of `registry` forces `request` to create a cursor.  Make sure @Julien00859 has a look at this.

Reply:
This is only to be able to use an ormcache on routing_map. When I tried to add the ormcache on a classmethod it failed.
But anyway, there is already an env when serving a database and thus a cursor on the request right? It doesn't look like a lazy property.